Services/Membership/API/CommandHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\API\Membership;
+
+use ILIAS\API as API;
+use \ILIAS\API\Exceptions as Exc;
+
+
+/**
+ * Command handler
+ *
+ * @author killing@leifos.de
+ */
+class CommandHandler extends API\Int\AbstractCommandHandler
+{
+	/**
+	 * @inheritdoc
+	 */
+	public function checkPolicyForCommand(API\Int\Command $command, int $actor_id): int
+	{
+		if ($actor_id === 7)
+		{
+			return self::POLICY_OK;
+		}
+
+		return self::POLICY_FAILED;
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function handle(API\Int\Command $command, int $actor_id)
+	{
+		if ($command instanceof AddCommand)
```

Comment:
I think we'll most propably need to know something about the parent(s) (or certain information of this/them) in the current execution chain here. In the concrete case we maybe need to distinguish a course parent from group parent.